docsite/rst/intro_configuration.rst

```diff
@@ -70,7 +70,7 @@ Actions are pieces of code in ansible that enable things like module execution,
 This is a developer-centric feature that allows low-level extensions around Ansible to be loaded from
 different locations::
 
-   action_plugins = ~/.ansible/plugins/action_plugins/:/usr/share/ansible_plugins/action_plugins
+   action_plugins = ~/.ansible/plugins/action:/usr/share/ansible/plugins/action
```

Comment:
these changes are not needed, names are actually arbitrary here.

the one thing it does reinforce is the usage of `x_plugins` which is still the way playbook/role plugin directories are named